package.json

```diff
@@ -48,9 +54,10 @@
     "start:regtest": "BSK_SUBDOMAIN_CONFIG=config-develop.json BSK_SUBDOMAIN_REGTEST=true npm run start",
     "build": "babel src -d lib",
     "flow": "flow",
+    "tsc": "tsc",
```

Comment:
as this is part of the build process, maybe it makes sense to rather edit the build command to be `tsc && babel src -d lib`?